openedx/frontend-app-authoring
# feat: Enable the Video editor in content libraries [FC-0062]

## Description

This enables the video editor in content libraries. For now it only has basic functionality, i.e. works with YouTube URLs. No transcript support. I'm not sure if other types of videos may work or not.

## Supporting information

Part of https://github.com/openedx/frontend-app-course-authoring/issues/1092

Private ref: [FAL-3827](https://tasks.opencraft.com/browse/FAL-3827)

## Testing instructions

Create a new video in a course.
Create a new video in a library.
Edit an existing video in a course. Ensure there are no regressions.
Edit an existing video in a library.
